### ISSUE TITLE
[Question]Quantization- export to ONNX: INVALID_GRAPH : This is an invalid model

### ISSUE THREAD

**Issue by Slyne** [NVIDIA/NeMo#1954]
**Describe your question**
```python
python examples/asr/quantization/speech_to_text_quant_infer.py --asr_model=QuartzNet15x5Base-En-max-256.nemo --dataset=/raid/data/data_librispeech/manifest.sample --onnx
```
The error:
```
W0323 03:12:57.813599 140599427417920 tensor_quantizer.py:280] Use Pytorch's native experimental fake quantization.
[NeMo W 2021-03-23 03:12:57 nemo_logging:349] /opt/conda/lib/python3.8/site-packages/torch/onnx/utils.py:1169: UserWarning: No names were found for specified dynamic axes of provided input.Automatically generated names will be applied to each dynamic axes of input audio_signal
      warnings.warn('No names were found for specified dynamic axes of provided input.'
    
[NeMo W 2021-03-23 03:12:57 nemo_logging:349] /opt/conda/lib/python3.8/site-packages/torch/onnx/utils.py:1169: UserWarning: No names were found for specified dynamic axes of provided input.Automatically generated names will be applied to each dynamic axes of input logprobs
      warnings.warn('No names were found for specified dynamic axes of provided input.'
    
[NeMo W 2021-03-23 03:12:58 nemo_logging:349] /opt/conda/lib/python3.8/site-packages/pytorch_quantization-2.1.0-py3.8-linux-x86_64.egg/pytorch_quantization/nn/modules/tensor_quantizer.py:285: TracerWarning: Converting a tensor to a Python number might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
      inputs, amax.item() / bound, 0,
    
[NeMo W 2021-03-23 03:12:58 nemo_logging:349] /opt/conda/lib/python3.8/site-packages/pytorch_quantization-2.1.0-py3.8-linux-x86_64.egg/pytorch_quantization/nn/modules/tensor_quantizer.py:291: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
      quant_dim = list(amax.shape).index(list(amax_sequeeze.shape)[0])
    
Warning: Constant folding supported for only opsets 9, 10, and 11. Constant folding not applied.
[W] Inference failed. You may want to try enabling partitioning to see better results. Note: Error was:
[ONNXRuntimeError] : 10 : INVALID_GRAPH : This is an invalid model. Type Error: Type 'tensor(float)' of input parameter (3317) of operator (DequantizeLinear) in node (DequantizeLinear_2326) is invalid.
```

QuartzNet15x5Base-En-max-256.nemo is from:
```python
 python speech_to_text_calibrate.py --asr_model=QuartzNet15x5Base-En --dataset=/raid/data/data_librispeech/manifest.sample
```
Is this the right way to quantize asr model ? If it is, how can I fix this error ?  

**Environment overview (please complete the following information)**

 - Environment location: [Bare-metal, Docker, Cloud(specify cloud provider - AWS, Azure, GCP, Collab)]   Docker
 - Method of NeMo install: [pip install or from source]. Please specify exact commands you used to install.  ./reinstall.sh
 - If method of install is [Docker], provide `docker pull` & `docker run` commands used.
``` 
git clone https://github.com/NVIDIA/NeMo.git
cd NeMo
DOCKER_BUILDKIT=1 docker build -f ./Dockerfile .
```

**Additional context**

Example: GPU model

**Comment by okuchaiev** [comment]
@skyw  any thoughts?

**Comment by skyw** [comment]
It should be fixed in 21.03.
What failed is the checker when calling `torch.onnx.export`. It needs opset13 support which comes with Pytorch 1.8 release. 21.02 was based on a dev version of 1.8.

**Comment by Slyne** [comment]
thanks! Close this question.
